src/Misc/MiddleWare.cpp

```diff
@@ -1550,12 +2115,65 @@ static rtosc::Ports middwareSnoopPortsWithoutNonRtParams = {
         d.obj = impl.config;
         Config::ports.dispatch(chomp(msg), d);
         rEnd},
+    {"presets/copy:s:ss:si:ssi", 0, 0,
+        [](const char *msg, rtosc::RtData &d) {
+            MiddleWareImpl *impl = (MiddleWareImpl*)d.obj;
+            d.obj = (void*)impl->parent;
+            MiddleWare &mw = *impl->parent;
+            assert(d.obj);
+
+            std::string args = rtosc_argument_string(msg);
+            d.reply(d.loc, "s", "clipboard copy...");
+
+            std::string url = rtosc_argument(msg, 0).s;
+            void* obj = impl->obj_store.get(url);
```

Comment:
ok, the impl->obj_store.get() line makes sense for why this code was transferred to this file.